src/layers/legacy/medVtkDataMeshBase/vtkMetaDataSet.h

```diff
@@ -99,16 +97,13 @@ class MEDVTKDATAMESHBASE_EXPORT vtkMetaDataSet: public vtkDataObject
   */
   template <class type> inline bool GetMetaData (std::string key, type &ret)
   {
-
-    //type toret;//=vtkMetaDataSet::VTK_META_UNKNOWN; // to be implemanted with good template
-    bool valid = itk::ExposeMetaData<type>(this->MetaDataDictionary, key, ret);
-    //ret = toret;
-    if(!valid)
-      return false;
-    return true;
+      if(!itk::ExposeMetaData<type>(this->MetaDataDictionary, key, ret))
```

Comment:
Base code are not good
But correction are not really better
The good way is:
`return itk::ExposeMetaData<type>(this->MetaDataDictionary, key, ret)`

Reply:
indeed ^^

Reply:
ok